reporting/sending_metrics_report.md

```diff
@@ -0,0 +1,19 @@
+# Sending Metrics Report
+
+## Configurations
```

Comment:
@dbaluku add an overview section which describes the purpose of the analytics report 

The analytics usage report is a monthly that is submitted to understand the utilization and usage of UgandaEMR across health facilities to provide data to drive adoption. This report provides a starting point to help support upgrades, and will replace EMR rollout data calls as the data is submitted on a regular basis

Reply:
Done